schemas/get-credentials-response.schema.json

```diff
@@ -11,8 +11,12 @@
         },
         "type": "array"
       },
-      "description": "The headers containing credentials.",
+      "description": "The headers containing credentials which tools should add to all requests to the URI.",
       "type": "object"
+    },
+    "expires": {
+      "description": "The time when the credentials expire and the tool should re-validate them, formatted as ISO 8601.\n\nIf set, tools **SHOULD** not invoke the `get` command of the Credential Helper for the URI before this time has passed or before the remote service indicates the credentials are expired, revoked, or invalid for any other reason. Otherwise, tools **MAY** assume that the credentials are valid indefinitely or for a tool-specified duration before re-validating them.",
```

Comment:
What about the corner case where the credentials (are expected to) expire in the middle of an operation?

Reply:
+1, recommend adding an "unless ..." clause in here.

Reply:
Added something to indicate that revalidating if the server returns a permission error is always ok.

Reply:
Also added something to allow tools to refresh early if they expect the credentials to expire mid-operation. It's up to the server to decide what to do if they do expire mid-operation